Add Troubleshooting entry about deleted clients (#1421)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/troubleshooting.adoc

This issue is resolved by clearing the cache and reloading the page.
In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].

== Registering a traditional client as Salt minion after deleting it

> This is not a valid scenario. Normally you migrate a traditional client to a Salt minion without deleting the client.

Reviewer: ```suggestion
It is recommended to migrate a traditional client to a Salt client directly without deleting it first.
For more information about this migration procedure, see xref:xxx

If you deleted the traditional client by accident, proceed as follows:
```

> Salt automatically detects that you have a traditional client and does the necessary changes itself. But if you somehow
> deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the
> client before registering it as Salt minion:

Reviewer: ```suggestion
////
Salt automatically detects that you have a traditional client and does the necessary changes itself.
But if you somehow deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the client before registering it as Salt minion:
////
```
I think we do not need the above text.  But keeping for reference for the moment, commented.

> 

Reviewer: ```suggestion
.Procedure: Registering traditional client as a Salt client
```

> . Remove the following file:

Reviewer: ```suggestion
. On the client, remove the following file:
```

+
----
/etc/sysconfig/rhn/systemid
----
+

> . Remove the following package:

Reviewer: ```suggestion
. On the client, remove the [package]``zypp-plugin-spacewalk`` package:
```

+
----

> zypp-plugin-spacewalk

Reviewer: ```suggestion
zypper rm zypp-plugin-spacewalk
```

----

> +

Reviewer: ```suggestion
. Register the client now as a Salt client.
  For more information, see xref:xxx
```

This issue is resolved by clearing the cache and reloading the page.
In most browsers, you can do this quickly by pressing kbd:[Ctrl+F5].

== Registering a traditional client as Salt minion after deleting it

It is recommended to migrate a traditional client to a Salt client directly without deleting it first.

> For more information about this migration procedure, see xref:xxx

Reviewer: ```suggestion
Salt detects that you have a traditional client and migrates it to the Salt contact method automatically.
```
I could find this describe elsewhere in our documentation.  Thus no xref: here.

If you deleted the traditional client by accident, proceed as follows:

> ////
> Salt automatically detects that you have a traditional client and does the necessary changes itself.
> But if you somehow deleted the traditional client and want to register it as a Salt minion again, you have to do the following steps on the client before registering it as Salt minion:
> ////

Reviewer: ```suggestion
```
I think we can remove this now.

Author: Agreed.

.Procedure: Registering traditional client as a Salt client
. On the client, remove the following file:
+
----
/etc/sysconfig/rhn/systemid
----
+
. On the client, remove the [package]``zypp-plugin-spacewalk`` package:
+
----
zypper rm zypp-plugin-spacewalk
----
. Register the client now as a Salt client.

>   For more information, see xref:xxx

Reviewer: ```suggestion
  For more information, see xref:client-configuration:registration-methods-saltbundle.adoc[].
```
